packages/zql/src/query/named.ts

```diff
@@ -256,13 +256,13 @@ export function withValidation<
             ReadonlyArray<ReadonlyJSONValue>,
             TReturnQuery
           >
-        )(context, ...validator(...args))) as ValidatedSyncedQueryWithContext<
+        )(context, ...validator(args))) as ValidatedSyncedQueryWithContext<
         TContext,
         TReturnQuery
       >;
     }
     return ((...args) =>
-      fn(...validator(...args))) as ValidatedSyncedQuery<TReturnQuery>;
+      fn(...validator(args))) as ValidatedSyncedQuery<TReturnQuery>;
```

Comment:
For the validator, it would be great to support https://github.com/standard-schema/standard-schema